nntrainer/src/fc_layer.cpp

```diff
@@ -167,44 +160,10 @@ void FullyConnectedLayer::copy(std::shared_ptr<Layer> l) {
 }
 
 Tensor FullyConnectedLayer::backwarding(Tensor derivative, int iteration) {
-  Tensor djdb;
-  Tensor y2 = derivative;
-  Tensor y;
-
-  if (!last_layer) {
-    djdb = derivative.multiply(hidden.apply(activation_prime));
-  } else {
```

Comment:
I am not sure, but where does these cases go?

Reply:
Please refer to below comment :)